packages/ui/src/models/visualization/flows/support/camel-component-schema.service.ts

```diff
@@ -70,6 +72,32 @@ export class CamelComponentSchemaService {
     }
   }
 
+  static getTooltipContent(camelElementLookup: ICamelElementLookupResult): string {
+    if (camelElementLookup.componentName !== undefined) {
+      const catalogLookup = CamelCatalogService.getCatalogLookup(camelElementLookup.componentName);
+      if (catalogLookup.catalogKind === 'component') {
```

Comment:
Can we use the `enum` instead? This will help to quickly check where do we have such conditions